app/controllers/transactions_controller.rb

```diff
@@ -1,4 +1,5 @@
 class TransactionsController < ApplicationController
+  before_action :find_transaction, only: %i[update destroy edit]
   def new
```

Comment:
@marderer let's keep empty line between `before_action` and and controller's actions